grafana-annotator/templates/secret.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: {{ include "grafana-annotator.fullname" $ }}-token
+  labels:
+    app: {{ include "grafana-annotator.name" $ }}
+    chart: {{ include "grafana-annotator.chart" $ }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
Will this just always be `Helm`? https://helm.sh/docs/chart_template_guide/builtin_objects/

```
Release.Service: The service that is rendering the present template. On Helm, this is always Helm.
```

Reply:
i can just remove

Reply:
I mean it's totally fine - just was curious about it